operator/v1/stable.ingresscontroller.testsuite.yaml

```diff
@@ -13,3 +13,406 @@ tests:
+      kind: IngressController
+      metadata:
+        name: default-unclosed-braces
+        namespace: openshift-ingress-operator
+      spec:
+        httpHeaders:
+          actions:
+            request:
+            - name: Content-Location
+              action:
+                type: Set
+                set:
+                  value: /my-first-blog-post
+            - name:   Content-Language
+              action:
+                type: Delete
+            - name: expires
+              action:
+                type: Set
+                set:
+                  value: "%[req.hdr(host),lower"
+    expectedError: "Either the header value provided is not in correct format or the sample fetcher/converter specified is not allowed. The dynamic header value will be interpreted as an HAProxy format string as defined in https://cbonte.github.io/haproxy-dconv/2.4/configuration.html#8.2.4 and may use HAProxy's %[] syntax and otherwise must be a valid HTTP header value as defined in https://datatracker.ietf.org/doc/html/rfc7230#section-3.2. Sample fetchers allowed are req.hdr, ssl_c_der. Converters allowed are lower, base64."
+  - name: Should not allow to set dynamic response header values with not allowed sample fetchers.
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: IngressController
+      metadata:
+        name: default-not-allowed-values
+        namespace: openshift-ingress-operator
+      spec:
+        httpHeaders:
+          actions:
+            response:
+            - name: X-Frame-Options
+              action:
+                type: Set
+                set:
+                  value: DENY      
+            - name: X-Target 
+              action:
+                type: Set
+                set:
+                  value: "%[req.hdrs(host),lower]"
+    expectedError: "Either the header value provided is not in correct format or the sample fetcher/converter specified is not allowed. The dynamic header value will be interpreted as an HAProxy format string as defined in https://cbonte.github.io/haproxy-dconv/2.4/configuration.html#8.2.4 and may use HAProxy's %[] syntax and otherwise must be a valid HTTP header value as defined in https://datatracker.ietf.org/doc/html/rfc7230#section-3.2. Sample fetchers allowed are res.hdr, ssl_c_der. Converters allowed are lower, base64."
+  - name: Should not allow empty value in response.
+    initial: |
+      apiVersion: operator.openshift.io/v1
+      kind: IngressController
+      metadata:
+        name: default-not-allowed-values
+        namespace: openshift-ingress-operator
+      spec:
+        httpHeaders:
+          actions:
+            response:
+            - name: X-Frame-Options
+              action:
+                type: Set
+                set:
+                  value:
+    expectedError: "some validation rules were not checked because the object was invalid; correct the existing errors to complete validation"
```

Comment:
Can you check the rest of the error message from this test, I don't think this is the part of the error message you care about, I would expect something along the lines of a `field.Required` error preceding this one